src/components/ProductRouter.js

```diff
@@ -1,27 +1,29 @@
 import React from 'react';
 import { useParams, Redirect } from 'react-router-dom';
-
+import Product from './Product'; // Importuj komponent Product
 import products from '../products.json';
 
-const Product = () => {
-	const { id, alias } = useParams();
+const ProductRouter = () => {
+	const { id } = useParams();
 
 	const [product = null] = products.filter((prod) => prod.id === parseInt(id));
 
 	if (product === null) {
 		return <Redirect to='/404.html' />;
 	}
+
 	const { name, description, category, price } = product;
+
+	
 	return (
-		<article
-			data-id={id}
-			data-alias={alias}
-		>
-			<h1>Produkt: {name}</h1>
-			<p>Kategoria: {category}</p>
-			<p>Opis: {description}</p>
-			<p>Cena: {price}</p>
-		</article>
+		<Product
```

Comment:
Jeśli chciałbyś wyświetlać w różny sposób dane to możesz użyć dodatkowego props-a, który określi jakie dane mają być pokazane np. `<Product version="small" ... />` 

PS. W podobny sposób zrobić z linkowaniem (o tym w innym komentarzu).